bucket/clamav.json

```diff
@@ -0,0 +1,51 @@
+{
+    "version": "0.102.4",
+    "description": "ClamAV is an open source antivirus engine for detecting trojans, viruses, malware & other malicious threats.",
+    "homepage": "https://www.clamav.net/",

```

Comment:
Slash is not needed

![image](https://user-images.githubusercontent.com/13260377/90260279-914ede80-de4b-11ea-848d-0ac6f2326a7d.png)